systemFlake.nix

```diff
@@ -5,14 +5,19 @@
 , supportedSystems ? flake-utils-plus.lib.defaultSystems
 , inputs
 
-, nixosConfigurations ? { }
 , sharedExtraArgs ? { }
-, nixosProfiles ? { }
 , channels ? { }
 , channelsConfig ? { }
 , sharedModules ? [ ]
 , sharedOverlays ? [ ]
 
+, nixosConfigurations ? { }
```

Comment:
```suggestion
```

This PR assumes resolution of #14 (see end of file)

Reply:
You can read my comment there for reasoning, but I would say drop `nixosConfigurations` entirely. It goes against the idea that flake-utils-plus is a tool.